nameko/testing/utils.py

```diff
@@ -89,6 +90,13 @@ def assert_stops_raising(fn, exception_type=Exception, timeout=10,
             eventlet.sleep(interval)
 
 
+MockCallArgs = namedtuple("MockCallArgs", ["positional", "keyword"])
+
+
+def unpack_mock_call(call):
```

Comment:
i had quick look in the source, and `Call` already has a bunch of interesting behaviours, e.g. it compares successfully to both a tuple (compared against `args`) and a dict (matched against `kwargz`) but nothing obvious to check membership in kwargs like you do

happy to keep this helper if you like it